contrib/buildsystems/Generators/Vcxproj.pm

```diff
@@ -139,7 +148,7 @@ sub createProject {
   </PropertyGroup>
   <PropertyGroup>
     <ConfigurationType>$config_type</ConfigurationType>
-    <PlatformToolset>v140</PlatformToolset>
+    <PlatformToolset>v142</PlatformToolset>
```

Comment:
Could we keep `v140` here to keep things working with VS 2015? Upon loading the solution in Visual Studio, the user is asked whether they want to retarget anyway, and even MSBuild can override this via `/p:PlatformToolset=v142`.